spring-boot-starters/langchain4j-open-ai-spring-boot-starter/src/main/java/dev/langchain4j/openai/spring/Properties.java

```diff
@@ -0,0 +1,26 @@
+package dev.langchain4j.openai.spring;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.NestedConfigurationProperty;
+
+@Getter
+@Setter
+@ConfigurationProperties(prefix = Properties.PREFIX)
+public class Properties {
+
+    static final String PREFIX = "langchain4j.open-ai";
+
+    @NestedConfigurationProperty
+    ChatModelProperties chatModel;
+
+    @NestedConfigurationProperty
+    LanguageModelProperties languageModel;
+
+    @NestedConfigurationProperty
+    EmbeddingModelProperties embeddingModel;
+
+    @NestedConfigurationProperty
+    ModerationModelProperties moderationModel;
+}
```

Comment:
Don't forget to add the new "Image" support that was merged today

Reply:
Good catch, thanks!